dts/arm/nxp/nxp_rt1064.dtsi

```diff
@@ -11,6 +11,7 @@
 	/* WINBOND */
 	w25q32jvwj0: w25q32jvwj@0 {
 		compatible = "winbond,w25q32jvwj", "jedec,spi-nor";
```

Comment:
Probably not a problem, but just because I noticed it: No 0 at the end of the compatible, unlike the other places.

Reply:
I suspect the 0 is not a part of the device model, but instead denotes an instance or CS index.  I don't see any way it would be a problem, since we don't have a binding yaml for that compatible.

Reply:
Yeah, typo at worst...